### ISSUE TITLE
GUI for service monitoring

### ISSUE THREAD

**Issue by vforv** [senecajs/seneca#666]
Is there any user interface to monitor senica services?

**Comment by kryptstorm** [comment]
We don't. What we have is a toolkit, I don't think we need a user interface now.

**Comment by rjrodger** [comment]
I'll be releasing https://github.com/rjrodger/seneca-monitor soon - it gives you a dynamic graph of the current network - is this what you mean?

**Comment by vforv** [comment]
Yes, that would be great. Also dash where we can see all services, messages, which services are up/down and how much instances currently running.

Thanks!

**Comment by snics** [comment]
@rjrodger does the seneca monitor actually work already? Looks like it does not work?
